user-guide-chapters/introduction/building.adoc

```diff
@@ -1,10 +1,79 @@
-== Building {zotLowerName} from Source
+== How to Get {zotLowerName}
 
+The {zotLowerName} project is hosted on GitHub at https://github.com/project-zot/zot[project-zot].
+From GitHub, you can download {zotLowerName} executable binary images or full source code.
 
 === Supported Platforms
 
-{zotUpperName} is officially supported on `Linux` and `Apple MacOS` platforms. However,
-development should be possible on any platform that supports the `golang` toolchain.
+{zotUpperName} is officially supported on Linux and Apple MacOS platforms, using
+Intel or ARM processors. However, development
+should be possible on any platform that supports the `golang` toolchain.
+
+[supported-platforms-and-architectures-table-zot]
+.Supported Platforms and Architectures
+[%autowidth]
+|===
+| OS | ARCH | Platform
+
+| linux | amd64 | Intel-based Linux servers
+| linux | arm64 | ARM-based servers and Raspberry Pi4
+| darwin | amd64 | Intel-based MacOS
+| darwin | arm64 | ARM-based MacOS
+|===
+
+An executable binary image for {zotLowerName} is named using the platform and architecture from the
+ <<supported-platforms-and-architectures-table-zot, Supported Platforms and Architectures>>
+ table. The format of an image name is:
+
+``zot-_os_-_architecture_``
+
+For example, the binary image for an Intel-based linux server is
+`zot-linux-amd64.`
+
+=== Deployment Methods
+
+Several options exist for deploying {zotLowerName}:
+
+- You can launch a {zotLowerName} binary as a container service using a container
+management tool such as Docker, Podman, or Helm.
+
+- You can launch {zotLowerName} as a host-level service by downloading a binary image
+and running it as a systemd service.
+
+- You can copy or clone the full {zotLowerName} source code and build an image
+with custom build flags.
+
+== Deploying a {zotLowerName} Binary Image
+
+Executable binary images for supported server platforms and architectures are
+available from the
+https://github.com/orgs/project-zot/packages?repo_name=zot[zot package repository]
+in GitHub.
+
+You can download the appropriate binary image and run it directly on your server, or
+you can use a container management tool such as Docker, Podman, or Helm to fetch
+and deploy the image in a container on your server.
+
+TIP: For convenience, you can rename the binary image file to simply `zot.`
+
+=== Example: Deploying with Docker
```

Comment:
I would like to have an example with podman/runc as well. And that should be the first one :)

Reply:
Added Podman example before Docker.